src/bionics.cpp

```diff
@@ -2717,8 +2715,10 @@ void Character::remove_bionic( const bionic_id &b )
     bionic_collection new_my_bionics;
     // any spells you should not forget due to still having a bionic installed that has it.
     std::set<spell_id> cbm_spells;
+    bool skipped_installed = false;
     for( bionic &i : *my_bionics ) {
-        if( b == i.id ) {
+        if( b == i.id && !skipped_installed ) {
+            skipped_installed = true;
```

Comment:
Is that meant to allow removing power storages ( or other duplicated CBMs) one at a time? Maybe putting a comment around here could clarify that